documentation/how-to/attributes-form.en.md

```diff
@@ -183,7 +183,11 @@ can be based on the geometry or expression variables.
 
 For more information regarding storing information related to your
 position in object attributes, refer to the deticated
-`GNSS documentation<gnss_variables>`{.interpreted-text role="ref"}.
+[GNSS documentation](./gnss/).
+
+For QFieldCloud users, two variables can be used with default values:
```

Comment:
Just default values or also other expressions?

Reply:
Certainly with other expressions, but this is mentioned within the default values section in the overall context of attribute tables, so I don't think it's appropriate to mention symbology here :)

Reply:
It currently gives a slight impression of only being usable with default values. Maybe "expressions like default values" or something the like could be used?

Reply:
What about: "For QFieldCloud users, two variables can be used in expressions including attribute form's default values:..." ?